crates/sncast/tests/data/files/correct_snfoundry.toml

```diff
@@ -33,6 +33,17 @@ account = "user1"
 wait-params = { timeout = 500, retry-interval = 10 }
 show-explorer-links = false
 
+[sncast.profile7]
+network = "sepolia"
+account = "user1"
+accounts-file = "/path/to/account.json"
+
 [sncast.no_url]
 accounts-file = "../account-file"
 account = "user1"
+
+[sncast.url_and_network]
+account = "user1"
+url = "http://some.url"
+network = "sepolia"
```

Comment:
Is having both correct? The validation we have for configs says it shouldn't be.